TypeCobol/Compiler/Diagnostics/TypeCobolChecker.cs

```diff
@@ -6,6 +6,7 @@
 using TypeCobol.Compiler.Parser;
 using TypeCobol.Compiler.Parser.Generated;
 using TypeCobol.Compiler.Scanner;
+using static TypeCobol.Compiler.Nodes.Node;
```

Comment:
I'm no big fan of `using static` because it makes harder to read wich method comes from which class. As it is used only for the `Flag` enum, remove this using and prefix the Flag name with `Node.`